Content.Shared/Pinpointer/SharedNavMapSystem.cs

```diff
@@ -34,6 +40,9 @@ public override void Initialize()
 
         // Data handling events
         SubscribeLocalEvent<NavMapComponent, ComponentGetState>(OnGetState);
+
+        SubscribeNetworkEvent<MapWarpRequest>(OnMapWarp);
+
```

Comment:
Just use SubscribeAll and it can be predicted?

Reply:
Done